src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/logic/WorkflowLogic.kt

```diff
@@ -34,18 +46,79 @@ class OrderlyWebWorkflowLogic : WorkflowLogic
         val columnCount = headers.count()
         val paramNames = headers.drop(1)
```

Comment:
I think it would help readability here to have a comment describing the expected headers; are they `report, param1, param2, ... paramN`?